crates/ruff_linter/src/rules/ruff/rules/parenthesize_logical_operators.rs

```diff
@@ -37,12 +40,20 @@ use crate::checkers::ast::Checker;
 pub struct ParenthesizeChainedOperators;
 
 impl Violation for ParenthesizeChainedOperators {
+    const FIX_AVAILABILITY: FixAvailability = FixAvailability::Sometimes;
+
     #[derive_message_formats]
     fn message(&self) -> String {
         format!(
             "Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear"
         )
     }
+
+    fn fix_title(&self) -> Option<String> {
+        Some(
+            "Put parentheses around the `and` subexpression inside the `or` expression".to_string(),
```

Comment:
We typically make this quite short since it shows up in the Code Action menu on VS Code -- I might recommend: "Add parentheses" or "Parenthesize the `and` expression".